app/src/main/java/com/kickstarter/viewmodels/ProjectCardHolderViewModel.java

```diff
@@ -194,9 +284,20 @@ public ViewModel(final @NonNull Environment environment) {
         .map(Project::stateChangedAt)
         .compose(coalesce(new DateTime()));
 
-      this.rootCategoryNameForFeatured = this.project
-        .map(Project::category)
-        .filter(ObjectUtils::isNotNull)
+      this.projectWeLoveIsGone = this.project
+        .map(Project::staffPick)
+        .compose(coalesce(false))
+        .compose(combineLatestPair(this.discoveryParams.map(DiscoveryParams::staffPicks).compose(coalesce(false))))
```

Comment:
Why is the `coalese(false)` opposed to taking in a value?

Reply:
`staffPick` is `@Nullable`. If it is null, we want to force it to be false.